package.json

```diff
@@ -23,6 +24,7 @@
         "prop-types": "^15.7.2",
         "react": "^18.0.0",
         "react-dom": "^18.0.0",
+        "react-hook-form": "^7.48.2",
```

Comment:
I think you can remove yup and rhf it was a mistake

Reply:
Ok apparently we can't compile without because it's in peer deps of commons ui, so just let it in dev deps